pkgs/development/python-modules/opensfm/default.nix

```diff
@@ -0,0 +1,111 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cmake
+, opencv
+, opencv4
+, ceres-solver
+, suitesparse
+, metis
+, eigen
+, pkg-config
+, pybind11
+, numpy
+, pyyaml
+, lapack
+, gtest
+, gflags
+, glog
+, pytestCheckHook
+, networkx
+, pillow
+, exifread
+, gpxpy
+, pyproj
+, python-dateutil
+, joblib
+, repoze_lru
+, xmltodict
+, cloudpickle
+, scipy
+, sphinx
+, matplotlib
+, fpdf
+}:
+
+let
+  glog' = glog.overrideAttrs (attrs: attrs // {
```

Comment:
`attrs //` can be removed because overriding is already doing set merging

Reply:
Thanks! I will remove the whole glog thing, I was just testing a hypothesis...